frontend/encore/dev-server.rst

```diff
@@ -66,53 +51,64 @@ method in your ``webpack.config.js`` file:
 
     The ``Encore.configureDevServerOptions()`` method was introduced in Encore 0.28.4.
 
-Hot Module Replacement HMR
---------------------------
+Enabling HTTPS using the Symfony Web Server
+-------------------------------------------
 
-Encore *does* support `HMR`_ for :doc:`Vue.js </frontend/encore/vuejs>`, but
-does *not* work for styles anywhere at this time. To activate it, pass the ``--hot``
-option:
+If you're using the :doc:`Symfony web server </setup/symfony_server>` locally with HTTPS,
+you'll need to also tell the dev-server to use HTTPS. To do this, you can reuse the Symfony web
+server SSL certificate:
+
+.. code-block:: javascript
+
+    // webpack.config.js
+    // ...
+
+    Encore
+        // ...
+
+    .configureDevServerOptions(options => {
+        options.https = {
+            pfx: '/path/to/.symfony/certs/default.p12',
```

Comment:
```suggestion
            pfx: path.join(process.env.HOME, '.symfony/certs/default.p12'),
```

What about this? Since it's the default Symfony local HTTPS certificat, I think it's a good, WDYT?

Reply:
(Note that is also require to import `path` module)